workflow/src/main/resources/queries/sd/aa

```diff
@@ -0,0 +1,23 @@
+-- BuildTextSearchInformation search-strings input query for procedures.
```

Comment:
is the filename "aa" a typo?

Reply:
Oops, removed. This was just a temp file.